src/Illuminate/Support/ServiceProvider.php

```diff
@@ -134,17 +134,19 @@ protected function loadMigrationsFrom($paths)
      * Register paths to be published by the publish command.
      *
      * @param  array  $paths
-     * @param  string  $group
+     * @param  string  $groups
```

Comment:
This should be `mixed` as `$groups` can be either a string, either an array.

Reply:
And in this case, it would be a breaking change for anyone extending this method, so this should target `master` instead

Reply:
Yep, you are right.

Should I redo this and make a PR to master instead?

Reply:
You can change the target branch in you PR